config/settings/base.py

```diff
@@ -477,10 +476,13 @@
 
 # The backend to be used by the ad server
 # for topic and keyword analysis
+# Set to `None` to disable the analyzer entirely
 ADSERVER_ANALYZER_BACKEND = env(
     "ADSERVER_ANALYZER_BACKEND",
     default="adserver.analyzer.backends.TextacyAnalyzerBackend",
```

Comment:
I wonder if we default this to None? Seems outside of scope for a lot of use cases.

Reply:
Ya, it could default to None. We might need to rework the test cases and how they're skipped but it would be OK.